frontend/src/components/common/image/image.tsx

```diff
@@ -23,6 +32,7 @@ const Image: FC<Props> = ({ width, height, src, alt, isCircular, classes }) => (
     height={height}
     src={src}
     alt={alt}
+    onClick={onClick}
```

Comment:
`img` is not an interactive element, can't have `onClick`

Reply:
agreed